projects/packages/connection/legacy/class-jetpack-ixr-client.php

```diff
@@ -122,4 +132,17 @@ public function get_jetpack_error( $fault_code = null, $fault_string = null ) {
 
 		return new \WP_Error( "IXR_{$fault_code}", $fault_string );
 	}
+
+	/**
+	 * Retrieve a response header if set.
+	 *
+	 * @param  string $name  header name.
+	 * @return string $value header value.
```

Comment:
This can return a string or bool, so maybe something like:

```suggestion
	 * @return string|bool Header value if set, false if not set.
```